src/engine/create-changeset/index.ts

```diff
@@ -26,20 +23,20 @@ export const createChangesetTask = (context: CreateChangesetContext): Listr => {
                 task: (ctx, task): Listr => {
                   return task.newListr(
                     [
-                      createEntitiesTask({
+                      createChangesetTasks.fetchPartialEntitiesTask({
```

Comment:
Really nice! We could even think of getting rid of the second `Task` everywhere, as it's already mentioned. So e.g. this could be 
```
createChangesetTasks.fetchPartialEntities()
```

WDYT?

Reply:
Same idea, complimentary suggestion 🙃